packages/react-app/src/pages/Help/index.tsx

```diff
@@ -103,6 +103,17 @@ const FAQ: React.FC = () => (
             </a>
           </p>
         </h2>
+        <h2 className={styles.questionHeader}>
+          Why do I get an error message when I try to load my map in
+          Civilization VI?
+        </h2>
+        <p className={styles.answerText}>
+          You may be experiencing a novel issue, but it is also likely that you
+          have added too many city states to your game. Currently the maps
+          generated here support only 1.5x the maximum number of civilizations
```

Comment:
Let's add the reason here: bug that prevents start locations from being generated on custom maps